src/pages/workspace/categories/WorkspaceCategoriesSettingsPage.tsx

```diff
@@ -45,20 +75,33 @@ function WorkspaceCategoriesSettingsPage({policy, route}: WorkspaceCategoriesSet
                 testID={WorkspaceCategoriesSettingsPage.displayName}
             >
                 <HeaderWithBackButton title={translate('common.settings')} />
-                <View style={styles.flexGrow1}>
-                    <ToggleSettingOptionRow
-                        title={translate('workspace.categories.requiresCategory')}
-                        subtitle={toggleSubtitle}
-                        switchAccessibilityLabel={translate('workspace.categories.requiresCategory')}
-                        isActive={policy?.requiresCategory ?? false}
-                        onToggle={updateWorkspaceRequiresCategory}
-                        pendingAction={policy?.pendingFields?.requiresCategory}
-                        disabled={!policy?.areCategoriesEnabled || !hasEnabledOptions || isConnectedToAccounting}
-                        wrapperStyle={[styles.mt2, styles.mh4]}
-                        errors={policy?.errorFields?.requiresCategory ?? undefined}
-                        onCloseError={() => Policy.clearPolicyErrorField(policy?.id ?? '-1', 'requiresCategory')}
-                        shouldPlaceSubtitleBelowSwitch
-                    />
+                <ToggleSettingOptionRow
+                    title={translate('workspace.categories.requiresCategory')}
+                    subtitle={toggleSubtitle}
+                    switchAccessibilityLabel={translate('workspace.categories.requiresCategory')}
+                    isActive={policy?.requiresCategory ?? false}
+                    onToggle={updateWorkspaceRequiresCategory}
+                    pendingAction={policy?.pendingFields?.requiresCategory}
+                    disabled={isToggleDisabled}
+                    wrapperStyle={[styles.pv2, styles.mh5]}
+                    errors={policy?.errorFields?.requiresCategory ?? undefined}
+                    onCloseError={() => Policy.clearPolicyErrorField(policy?.id ?? '-1', 'requiresCategory')}
+                    shouldPlaceSubtitleBelowSwitch
+                />
+                <View style={[styles.containerWithSpaceBetween]}>
+                    {!!currentPolicy && sections?.length > 0 && (
+                        <SelectionList
+                            headerContent={
+                                <View style={[styles.mh5, styles.mt2]}>
+                                    <Text style={[styles.headerText]}>{translate('workspace.categories.defaultSpendCategories')}</Text>
+                                    <Text style={[styles.mt1, styles.lh20]}>{translate('workspace.categories.spendCategoriesDescription')}</Text>
```

Comment:
I think `mv1` is better here to add more space after description

![Screenshot 2024-09-03 at 12 29 30 PM](https://github.com/user-attachments/assets/51b1f3d9-4775-4233-a81a-68c6923994b4)